docs/README.build.md

```diff
@@ -0,0 +1,24 @@
+# How to build snabb-lwaftr
+
+## Step 1: Fetch the sources
+
+```bash
+$ git clone https://github.com/Igalia/snabbswitch.git
+```
+
+## Step 2: Check out the lwaftr development branch:
+
+```bash
+$ cd snabbswitch && git checkout lwaftr_nutmeg
+```
+
+## Step 3: Build
+
+```bash
+$ make
```

Comment:
JFYI: You could also put the lwaftr code in `src/program/lwaftr` and then you would automatically get the Makefile target `make bin/lwaftr`. This way your code would all be compiled into the main binary instead of run as an external script. (If you have shell scripts you could also link those into the binary - see snabbnfv for an example.)

The main reason to do this would be to distribute your application as a single stand-alone file if you would prefer not to depend on the right version of a `snabb` binary to be installed separately.

(I am curious for feedback about what people like/dislike about this model.)